### PR TITLE
Fix git VSN on build release

### PR DESCRIPTION
**Pull request by kxepal**

### CODE REVIEW

**Comment by janl** [comment]
+1

File: build-aux/couchdb-build-release.sh [hunk 1]
```diff
@@ -23,6 +23,7 @@ cd src/
 
 for repo in *; do
   cd $repo
+  grep -rl '{vsn, git}' | xargs sed -i "s/{vsn, git}/{vsn, \"${VERSION}-dev\"}/"
```
Comment (rnewson) [inline build-aux/couchdb-build-release.sh]: can we grab the actual value here? For users of couchdb 2.0, 2.1, etc, they won't see or care about these values, but other users, like Cloudant, will make releases that only change some applications, so showing the actual git value will help. Specifically, the output of `git describe --always --tags` within that directory.

Comment (kxepal) [inline build-aux/couchdb-build-release.sh]: Sure. So: ${VERSION}-`git describe --always --tags` ?

Comment (rnewson) [inline build-aux/couchdb-build-release.sh]: hm, I think just the value of `git describe --always --tags`. the $VERSION value should be in the tarball file name and in the GET / welcome message.

Comment (kxepal) [inline build-aux/couchdb-build-release.sh]: ok

**Comment by kxepal** [comment]
@janl @rnewson updated PR with _correct_ vsn fix (:
I don't like `set +e` `set -e` around `grep ... | xargs sed ...`, but didn't found any clever way to not have script fail for apps which already provides not-git vsn. Thoughts?

**Comment by rnewson** [comment]
+1, we can always tidy later when inspiration strikes.

**Comment by kxepal** [comment]
Good. Landed. Thanks!

**Pull request merged**